src/components/learner-credit-management/BudgetDetailAssignments.jsx

```diff
@@ -57,32 +58,53 @@ const BudgetDetailAssignments = ({
 
   return (
     <section className="budget-detail-assignments">
-      <h3 className="mb-3" ref={assignedHeadingRef}>
-        <FormattedMessage
-          id="lcm.budget.detail.page.assignments.heading"
-          defaultMessage="Assigned"
-          description="Heading for the assigned section on the budget detail page"
-        />
-      </h3>
-      <p className="small mb-4">
-        <FormattedMessage
-          id="lcm.budget.detail.page.assignments.description"
-          defaultMessage="Assigned activity earmarks funds in your budget so you can not overspend.
+      {isRetired ? (
```

Comment:
[nit/suggestion] Might be worth splitting this out into its own component for a cleaner implementation

Example component name: `<BudgetDetailAssignmentsHeader/>`